### ISSUE TITLE
Blog: contrast, layout of buttons …

### ISSUE THREAD

**Issue by grahamperrin** [BrowserWorks/waterfox#664]
Meta, tracking: #582 

Suggested priority: P5

An unmodified view: 

![2018-06-18 05 25 13](https://user-images.githubusercontent.com/192271/41520462-b3c054ba-72c5-11e8-9f83-9b91a0fdda9a.png)

## Issues

Accessibility: buttons, and some of the texts in the left hand column, lack contrast. 

I'm not sure why one of the four buttons is to the right. 

### Workarounds 

I'll post to Reddit.

**Comment by grahamperrin** [comment]
## Contrast of texts in buttons

For one of the buttons: 

```text
  Info for current <a> element
  -------------------------------------- font ---------------------------------
  font size: 14px,
  font style: normal,
  font weight: 300,
  font variant: normal,
  rendered font family: sans-serif,
  font family: sans-serif,
  ------------------------------------- color ---------------------------------
  font color: rgb(155, 155, 155),
  background color: rgb(251, 251, 251),
  -------------------------------------- text ----------------------------------
  text align: start,
  text indent: 0px,
  line height: 24px,
  text transform: none,
  text decoration: none,
  ------------------------------------- HTML --------------------------------
  HTML element id: N/A,
  HTML element type: a,
  HTML element class: button,
  -------------------------------------- tab -----------------------------------
  tab.id: 605,
  tab.height: 751px,
  tab.width: 1240px,
  tab.title: The Waterfox Blog | Waterfox 56.2.1 Release,
  tab.url: https://blog.waterfoxproject.org/waterfox-56.2.1-release-download
```

– information presented by [Font Inspector](https://addons.mozilla.org/addon/font-inspector-webextension/).
